### PR TITLE
ref(test): Remove some uneeded jquery usage

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: tests/js/spec/components/dropdownLink.spec.jsx [hunk 1]
```diff
@@ -131,7 +131,14 @@ describe('DropdownLink', function () {
...
 
       it('does not close when document is clicked', function () {
-        $(document).click();
+        const event = document.createEvent('HTMLEvents');
+        event.initEvent('click', false, true);
+        document.dispatchEvent(event);
+
+        const evt = document.createEvent('HTMLEvents');
+        evt.initEvent('click', false, true);
+        document.body.dispatchEvent(evt);
```
Comment (priscilawebdev) [inline tests/js/spec/components/dropdownLink.spec.jsx:140]: these two blocks of code are very similar, except for `evt` and `event`. Should we remove one of them?

Comment (EvanPurkhiser) [reply tests/js/spec/components/dropdownLink.spec.jsx:140]: oops sorry, didn't push my fix 🙃

File: tests/js/spec/components/dropdownLink.spec.jsx [hunk 2]
```diff
@@ -131,7 +131,10 @@ describe('DropdownLink', function () {
       });
 
       it('does not close when document is clicked', function () {
-        $(document).click();
+        const evt = document.createEvent('HTMLEvents');
```
Comment (priscilawebdev) [inline tests/js/spec/components/dropdownLink.spec.jsx:134]: should we name this const event too? just to be consistent :D

**Review by priscilawebdev — APPROVED** [review]
lgtm

File: tests/js/spec/components/dropdownLink.spec.jsx (hunk 2 above)
Comment (priscilawebdev) [reply tests/js/spec/components/dropdownLink.spec.jsx:134]: nah...I just noticed that in this test everything is evt

Comment (EvanPurkhiser) [reply tests/js/spec/components/dropdownLink.spec.jsx:134]: yeah 🤷

**Pull request merged**